engine/src/main/java/org/terasology/network/internal/ClientConnectionHandler.java

```diff
@@ -69,13 +72,24 @@ public ClientConnectionHandler(JoinStatusImpl joinStatus, NetworkSystemImpl netw
         this.moduleManager = CoreRegistry.get(ModuleManager.class);
     }
 
-    private void scheduleTimeout(Channel inputChannel) {
+    private void scheduleTimeout(Channel inputChannel, NetData.NetMessage message) {
         channel = inputChannel;
         timeoutPoint = System.currentTimeMillis() + timeoutThreshold;
         timeoutTimer.schedule(new java.util.TimerTask() {
             @Override
             public void run() {
+                System.out.println("AAAAAAAAAAAAAAAAAAAAAAAAAA");
```

Comment:
I forgot to remove that debug statement before I committed. I'm using System,out.println() for debug statements because it is easier to notice and remove them before commiting (which I failed to do this time...).